src/main/java/club/sk1er/hytilities/handlers/lobby/LobbyChecker.java

```diff
@@ -20,25 +20,33 @@
 
 import club.sk1er.hytilities.Hytilities;
 import club.sk1er.mods.core.util.MinecraftUtils;
-import net.minecraftforge.client.event.ClientChatReceivedEvent;
-import net.minecraftforge.fml.common.eventhandler.EventPriority;
+import net.minecraft.client.Minecraft;
+import net.minecraftforge.event.world.WorldEvent;
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import net.minecraftforge.fml.common.gameevent.TickEvent;
 
 import java.util.regex.Pattern;
 
 public class LobbyChecker {
-
     private final Pattern lobbyPattern = Pattern.compile("(.+)?lobby(.+)");
+    private int tick;
     private boolean lobbyStatus;
 
-    @SubscribeEvent(priority = EventPriority.LOWEST)
-    public void retrieveLocraw(ClientChatReceivedEvent event) {
-        if (!MinecraftUtils.isHypixel() || Hytilities.INSTANCE.getLocrawUtil().getLocrawInformation() == null) {
-            this.lobbyStatus = false;
+    @SubscribeEvent
+    public void onTick(TickEvent.ClientTickEvent event) {
+        if (event.phase != TickEvent.Phase.START || Minecraft.getMinecraft().thePlayer == null || !MinecraftUtils.isHypixel() || this.tick >= 52 || Hytilities.INSTANCE.getLocrawUtil().getLocrawInformation() == null) {
             return;
         }
+        this.tick++;
+        if (this.tick == 50) {
```

Comment:
i believe this could be made `this.tick++ == 50` just to reduce the amount of code going on

Reply:
I believe it would need to be `++this.tick == 50` for the logic to be the same